src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -134,6 +140,20 @@ def get_app_data(*args):
     return {"info": {"name": APP_NAME}}
 
 
+def ts_are_close(t0: int, t1: int, tol: float = 1) -> bool:
```

Comment:
For when not mocking out `biokbase.narrative.jobs.jobcomm.time.time_ns` with `42`, check that `last_checked` ts is approximately right